subprojects/core/src/integTest/groovy/org/gradle/normalization/ConfigureRuntimeClasspathNormalizationIntegrationTest.groovy

```diff
@@ -222,6 +268,28 @@ class ConfigureRuntimeClasspathNormalizationIntegrationTest extends AbstractInte
             return this
         }
 
+        ProjectWithRuntimeClasspathNormalization withManifestAttributesIgnored() {
+            root.file('build.gradle') << """
+                normalization {
+                    runtimeClasspath {
+                        ignoreManifestAttribute "Implementation-Version"
+                    }
```

Comment:
That would be a useful shorthand. Let me see about that.

Reply:
Thinking about it - I think this makes more sense for this to be called `metaInf` instead of `manifest`. This started as being originally only about `META-INF/MANIFEST.MF` but properties filtering applies to `META-INF/*.properties` and you'd want `ignoreCompletely()` to ignore `META-INF/*` and `ignoreManifest()` to ignore `META-INF/MANIFEST.MF`.

Reply:
We already use `metaInf` in the `Jar` task itself, so yeah, I see how this would be useful. Good points about the `ignoreCompletely()` and `ignoreManifest()` methods, but let's leave those out of this PR.

I wonder if we want to specifically target `META-INF/*.properties` with this PR, though. We want to be able to filter any property in any `*.properties` file eventually. Maybe address that in a separate PR and leave the properties filtering out of this one? There is an added challenge about `*.properties` files, which is that their encoding is not predetermined. I imagine we'd want to do something like this:

```groovy
normalization {
    runtimeClasspath {
        properties("META-INF/*.properties") {
            encoding "UTF-8"
            ignoreProperty "..."
        }
    }
}
```

But if we go that way, this one can stay just `manifest { ... }`, no need to make it about `META-INF/`.

Reply:
The reason for this pull request is to allow us to adopt the build cache - without properties support this the normalization doesn't help us. Happy to leave out those convenience methods, but this is more about `META-INF`, not just the manifest attributes.

I believe properties files only support ISO 8859-1 encoding. Unicode characters are only supported by escapes?

Reply:
That's no longer the case: https://openjdk.java.net/jeps/226

Reply:
Maybe that's specifically to properties files loaded by `ResourceBundle`...

Reply:
Good to know - in that case I'll do what `PropertyResourceBundle` does - wire up an {{CharsetDecoder}} that attempts UTF-8 decoding by default and with a fallback to ISO 8859-1.